examples/fishjam-chat/screens/RoomScreen/RoomScreen.tsx

```diff
@@ -143,19 +141,7 @@ const RoomScreen = ({ navigation, route }: Props) => {
           onPress={onToggleScreenShare}
           accessibilityLabel={SHARE_SCREEN_BUTTON}
         />
-        {Platform.OS === 'ios' ? (
-          <InCallButton
-            iconName={
-              iosAppScreenShare?.isAppScreenShareOn
-                ? 'cellphone-screenshot'
-                : 'cellphone-off'
-            }
-            onPress={() => {
-              iosAppScreenShare?.toggleAppScreenShare();
-            }}
-          />
-        ) : null}
-
+        {Platform.OS === 'ios' && <ToggleAppScreenButton />}
```

Comment:
**nitpick:** I think it would be better if `ToggleAppScreenButton` handled this case. It's not possible to use it outside of iOS so I think it should return null for other platforms.

Reply:
I do not understand your point. `ToggleAppScreenButton` handles it, if some one use it he would get button which does nothing. Lets remember it is our example app.
Also from layout view it is more important to parent view to decide if child should be displayed than child decides for itself.